integration_tests/pages/assess/makeADecisionPage.ts

```diff
@@ -10,4 +10,8 @@ export default class MakeADecisionPage extends AssessPage {
   completeForm() {
     this.checkRadioButtonFromPageBody('decision')
   }
+
+  enterDecisionRationale(reason: string) {
```

Comment:
I think it would be best if we used `this.completeTextInputFromPageBody('decisionRationale')` here to follow the format that the other `AssessPage`s use